### PR TITLE
Do not validate Mandatory File in async call.

### PR DESCRIPTION
**Pull request by moumoutte**

### CODE REVIEW

File: formidable/views.py [hunk 1]
```diff
@@ -117,9 +118,24 @@ def get(self, request, format=None):
 
 
 class ValidateView(six.with_metaclass(MetaClassView, APIView)):
+    """
+    This view is usually called by the UI front-end in order to validate
+    data inside a form in order to avoid uploading file.
```
Comment (wo0dyn) [inline formidable/views.py]: Nitpick: s/in order to/to/ • This will avoid repetition in the sentence :wink:

File: formidable/views.py [hunk 2]
```diff
@@ -117,9 +118,24 @@ def get(self, request, format=None):
...
+    The main idea is to call this view in order to validate the entire data
+    except the File part. The UI front-end is not able to compute complex
+    validation (through validation presets), and to avoid heavy files exchanges
+    this view can validate first the data (without the file).
+
+    The final errors in the form are send to UI in order to display it.
```
Comment (wo0dyn) [inline formidable/views.py]: s/send/sent/

**Comment by moumoutte** [comment]
@wo0dyn Commit ammended

**Comment by moumoutte** [comment]
@wo0dyn : Commit ammended

**Pull request merged**